services/masterdata/src/assets/i18n/en.json

```diff
@@ -0,0 +1,172 @@
+{
+	"masterdatamgmt": {
+		"globals": {
+			"btnBack": "Back to list",
+			"btnCancel": "Cancel"
+		},
+		"pages": {
+			"dashboard": {
+				"demoText": "masterdata dashboard!"
+			},
+			"masterdataTypeList": {
+				"header": "Total masterdata types",
+				"deleteRowConfirmation": "Are you sure?",
+				"tableColumns": {
+					"code": "Code",
+					"name": "Name",
+					"count": "Count",
+					"description": "Description"
+				},
+				"buttons": {
+					"btnAddNewType": "Add masterdata type",
+					"btnView": "View Masterdatas",
+					"btnAdd": "Add Masterdata",
+					"btnEdit": "Edit",
+					"btnDelete": "Delete"
+				}
+			},
+			"masterdataTypeDetails": {
+				"header": "Masterdata-type Details",
+				"modelFields": {
+					"id": "Id",
+					"code": "Code",
+					"name": "Name",
+					"description": "Description",
+					"metadata": "Metadata",
+					"masterdatasCount": "Masterdatas Count"
```

Comment:
Shouldn't this be Master data item count?

Reply:
I prefer it like "masterdatas"

Reply:
'data', as in the english word, by nature is plural. so the 's' behind it doesn't make sense (which for other words is used to make them plural). So here it should be 'Masterdata count'.